cluster/aws/util.sh

```diff
@@ -721,14 +723,11 @@ function create-iam-profile {
 
 # Creates the IAM roles (if they do not already exist)
 function ensure-iam-profiles {
-  aws iam get-instance-profile --instance-profile-name ${IAM_PROFILE_MASTER} || {
```

Comment:
Before the cluster comes up, the new function `delete-iam-profiles` is run to ensure any pre-existing iam roles/profiles/relationships are cleaned up before this point.